src/mesh/RadioInterface.cpp

```diff
@@ -546,7 +551,13 @@ void RadioInterface::applyModemConfig()
     slotTimeMsec = computeSlotTimeMsec(bw, sf);
     preambleTimeMsec = getPacketTime((uint32_t)0);
     maxPacketTimeMsec = getPacketTime(meshtastic_Constants_DATA_PAYLOAD_LEN + sizeof(PacketHeader));
-
+#if defined(M5STACK_COREBASIC)   || defined(M5STACK_CORE2)
+    DataRegion.lora_channel_num=channel_num;
+    DataRegion.lora_freq=getFreq();
+    DataRegion.lora_channel_name=channelName;
+    DataRegion.lora_power_output=power;
+    DataRegion.lora_bw=bw;
+#endif
     LOG_INFO("Radio freq=%.3f, config.lora.frequency_offset=%.3f\n", freq, loraConfig.frequency_offset);
```

Comment:
See above. The DataRegion struct and drawLoraMessage are a new feature and belong in a separate PR.